### DISCUSSION TITLE
The direction of the arrow in the workflow.

### DISCUSSION THREAD

**Discussion by miranshub** [pcottle/learnGitBranching#1192] [Ideas]
In the playground, the direction of the arrow seems to be reversed, as we commit in c0, the the arrow is directed from c1 to c0 rather than from c0 to c1.
As per my suggestion this must be changed and the direction of the arrows must follows the direction towards the new commits like **_c0->c1->c2_**, rather than **c0<-c1<-c2**
![image](https://github.com/user-attachments/assets/ad43adce-3744-4520-b89e-6bbbac1b0636)

**Comment by Bubka** [comment]
Hi,
As explained in #992, this reflects how git works.

<sup>Originally posted by **mgarciaisaia** :</sup>
> That's how git really works. `C0` doesn't know a thing about `C1` - it's `C1` that has a pointer to its parent commit.

**Comment by pcottle** [comment]
Yep as @Bubka said above. There's a command to flip the arrow direction if you want, called flip tree I think -- but I still think this is a better default direction since it reflects how git works internally 😇
